code/go/0chain.net/conductor/config/types.go

```diff
@@ -40,6 +40,25 @@ func (n *LockNotarizationAndSendNextRoundVRF) Decode(val interface{}) error {
 	return mapstructure.Decode(val, n)
 }
 
+// Hardfork represents the hardfork directive state.
+type Hardfork struct {
+	Name string `json:"name" yaml:"name" mapstructure:"name"`
+}
+
+func (hf *Hardfork) IsValid() error {
+	validHardforks := map[string]bool{
```

Comment:
this creates hidden coupling. We need to add a field here every time a new fork is created. No need to check the validity since we ourselves will be writing these tests anyways.